CMakeLists.txt

```
@@ -1,6 +1,7 @@
 cmake_minimum_required(VERSION 2.6)
 
 option(VITA_BUILD "Build for the PlayStation Vita handheld game console." OFF)
+option(SWITCH_BUILD "Build for the Nintendo Switch handheld game console." OFF)
```

```suggestion
cmake_dependent_option(SWITCH_BUILD "Build for the Nintendo Switch handheld game console." OFF "NOT VITA_BUILD" OFF)
```

Since they can't be built in the same CMake configuration. And maybe add a condition to display an error if both `VITA_BUILD` and `SWITCH_BUILD` are set to `ON`?

---

fixed in 
https://github.com/bvschaik/julius/pull/124/commits/4e3c8b084b64d8ccde179fe6d8cfb3027195920c with other improvements (see commit)